lib/resque/failure.rb

```diff
@@ -31,6 +31,7 @@ def self.backend=(backend)
     # Returns the current backend class. If none has been set, falls
     # back to `Resque::Failure::Redis`
     def self.backend
+      @backend ||= nil
```

Comment:
Same comment as the `@heartbeat_thread` variable. Is it possible that we move this initialization to somewhere more obvious?